### ISSUE TITLE
Age Buckets

### ISSUE THREAD

**Issue by numb3r33** [numb3r33/airbnb_bookings#2]
Dataset representing information regarding various age buckets. 

See how can we merge this data with our training set and test set and use the information to create new features.
